src/group_theory/subgroup.lean

```diff
@@ -363,6 +363,14 @@ attribute [norm_cast] coe_Inf add_subgroup.coe_Inf
 @[simp, to_additive]
 lemma mem_Inf {S : set (subgroup G)} {x : G} : x ∈ Inf S ↔ ∀ p ∈ S, x ∈ p := set.mem_bInter_iff
 
+@[to_additive]
+lemma mem_infi {ι : Sort*} {S : ι → subgroup G} {x : G} : (x ∈ ⨅ i, S i) ↔ ∀ i, x ∈ S i :=
+by simp only [infi, mem_Inf, set.forall_range_iff]
+
+@[simp, to_additive]
+lemma coe_infi {ι : Sort*} {S : ι → subgroup G} : (↑(⨅ i, S i) : set G) = ⋂ i, S i :=
+by simp only [infi, coe_Inf, set.bInter_range]
+
```

Comment:
Please add `attribute [norm_cast]` to `coe_infi`, `add_subgroup.coe_infi`, `add_submonoid.coe_Inf`, and `add_submonoid.coe_infi`.
Otherwise LGTM.
bors d+

Reply:
Added, good find